test/integration/common.go

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
The federation integration fixture already has a `ClusterNames` method that returns the same information as the method in this file, as does the e2e framework.

Reply:
Updates to use the already existing function.